include/alpaka/core/Common.hpp

```diff
@@ -76,7 +76,21 @@
 //! In contrast to ordinary variables, you can not define such variables
 //! as static compilation unit local variables with internal linkage
 //! because this is forbidden by CUDA.
-#if(BOOST_LANG_CUDA && BOOST_ARCH_PTX) || (BOOST_LANG_HIP && (BOOST_ARCH_HSA || BOOST_ARCH_PTX))
+//!
+//! \attention It is not allowed to initialize the variable together with the declaration.
```

Comment:
Should be `definition`, not `declaration`? It is generally not allowed at declarations that are not definitions

Reply:
Same is also below

Reply:
updated